platforms/web/src/containers/RoutesContainer/RoutesContainer.tsx

```diff
@@ -5,20 +5,17 @@ import { useTranslation } from 'react-i18next';
 import { useProfileStore } from '@jwp/ott-common/src/stores/ProfileStore';
 import { useProfiles } from '@jwp/ott-hooks-react/src/useProfiles';
 import { useAccountStore } from '@jwp/ott-common/src/stores/AccountStore';
-import { PATH_USER_PROFILES } from '@jwp/ott-common/src/paths';
 import { shallow } from '@jwp/ott-common/src/utils/compare';
 import LoadingOverlay from '@jwp/ott-ui-react/src/components/LoadingOverlay/LoadingOverlay';
-import useQueryParam from '@jwp/ott-ui-react/src/hooks/useQueryParam';
 
 import useNotifications from '#src/hooks/useNotifications';
 
 const RoutesContainer = () => {
   const { i18n } = useTranslation();
   const location = useLocation();
-  const userModal = useQueryParam('u');
 
   const { profile, selectingProfileAvatar } = useProfileStore();
-  const { shouldManageProfiles } = useProfiles();
+  useProfiles();
```

Comment:
Can we fully remove them from the repo?

Reply:
and the line above should be removed as well :)

Reply:
@borkopetrovicc I think we also have a service and a controller for profiles.

Reply:
@AntonLantukh I know, but as we discussed, we will just remove the visual flow for now. In Q3, we will make an additional effort for complete removal

Reply:
@darkoatanasovski I think visual part is still there:

<img width="651" alt="Screenshot 2024-06-06 at 13 33 48" src="https://github.com/jwplayer/ott-web-app/assets/27254986/8feae508-4521-4c02-969e-c1ca5b4db0f5">

Reply:
a2246a4929c146792113f23c8a52749973cd267f